python/kfserving/test/test_storage.py

```diff
@@ -20,20 +20,6 @@
 
 STORAGE_MODULE = 'kfserving.storage'
 
-
-def test_storage_local_path():
-    abs_path = 'file:///'
-    relative_path = 'file://.'
-    assert kfserving.Storage.download(abs_path) == abs_path.replace("file://", "", 1)
-    assert kfserving.Storage.download(relative_path) == relative_path.replace("file://", "", 1)
-
-
-def test_storage_local_path_exception():
-    not_exist_path = 'file:///some/random/path'
-    with pytest.raises(Exception):
-        kfserving.Storage.download(not_exist_path)
-
-
 def test_no_prefix_local_path():
```

Comment:
Do you mind renaming this test to "test_local_path"

Reply:
> I'm not sure I understand this go portion of this PR. When I said local development story, I was thinking about running the model server locally, nothing about the controller.

Not sure whether I understand correctly. Do you mean to run kfserver directly without k8s or even docker but from python command line? Then I think it's always supported and no need to specify storageUri from spec since we even don't need the spec for k8s right?
What this PR fixed is the file:// type storageUri does not work. The "file://" type here I thought is for local development in which user build the kfserver by themselves and build the model together with the kfserver in the same image so that no need to download the model from s3, gs or even pvc since it's in the same context with the kfserver. (if it's not for this story, then what's the real meaning for this file://? That's why I want to delete it at beginning because it does not work for a long time)
There's some issues like item #567 for this, and I am curious about it too, so suggest we either fix it or remove it.

Reply:
> Do you mind renaming this test to "test_local_path"

I think after this fix, the "file://" type storageUri will not go through the storage initializer and there's no chance to download it this way, so need not keep the test for nothing.

Reply:
So to be honest I don't know what the local development really means and what the use case is intended for this "file://", just it's really a bug because in all the ways it does not work for now.